src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.styles.ts

```diff
@@ -0,0 +1,130 @@
+import {
+    FontSizes,
+    FontWeights,
+    IStyle,
+    ITheme,
+    mergeStyleSets
+} from '@fluentui/react';
+import {
+    IConditionsCalloutStyleProps,
+    IConditionsCalloutStyles
+} from './ConditionsCallout.types';
+
+export const classPrefix = 'cb-conditionscallout';
+const classNames = {
+    root: `${classPrefix}-root`,
+    footer: `${classPrefix}-footer`,
+    title: `${classPrefix}-title`
+};
+export const getStyles = (
+    _props: IConditionsCalloutStyleProps
+): IConditionsCalloutStyles => {
+    return {
+        root: [classNames.root],
+        footer: [
+            classNames.footer,
+            {
+                paddingTop: 32
+            }
+        ],
+        title: [
+            classNames.title,
+            {
+                fontSize: FontSizes.size16,
+                margin: 0
+            }
+        ],
+        subComponentStyles: {
+            callout: {
+                root: {
+                    width: 300,
+                    padding: 20
+                }
+            }
+        }
+    };
+};
+
+export const getSummaryStyles = (theme: ITheme) =>
```

Comment:
can we please put the styles for each component in their own file so they are easy to find? We follow that pattern across teh repo and it's good to stay consistent.